.github/actions/upload-alerts/action.yml

```diff
@@ -22,15 +22,14 @@ runs:
 
     - name: Run Script to Validate Alerts
       shell: bash
-      if: github.event_name != 'pull_request'
       run: | 
-        python tools/scripts/validate_alerts.py --alerts "${{ inputs.alerts }}"
+        python _test-infra/tools/scripts/validate_alerts.py --alerts '${{ inputs.alerts }}'
 
     - name: Run Script to Upload Alerts
       shell: bash
       if: github.event_name != 'pull_request'
       run: |
-       python tools/scripts/upload_alerts_to_aws.py "${{ inputs.alerts }}" "${{ inputs.organization }}" "${{ inputs.repo }}"
+       python _test-infra/tools/scripts/upload_alerts_to_aws.py --alerts '${{ inputs.alerts }}' --org "${{ inputs.organization }}" --repo "${{ inputs.repo }}"
```

Comment:
Just curious, I guess you need to use single quote here to not break up the JSON structure in `inputs.alerts`, right?